tests/unit/lms/views/api/canvas/authorize_test.py

```diff
@@ -98,31 +89,42 @@ def test_it_includes_the_state_in_a_query_param(
             canvas_oauth_callback_schema.state_param.return_value
         ]
 
-    def assert_sections_scopes(self, response):
-        query_params = parse_qs(urlparse(response.location).query)
-        assert query_params["scope"] == [
-            "url:GET|/api/v1/courses/:course_id/files "
-            "url:GET|/api/v1/files/:id/public_url "
-            "url:GET|/api/v1/courses/:id "
-            "url:GET|/api/v1/courses/:course_id/sections "
-            "url:GET|/api/v1/courses/:course_id/users/:id"
-        ]
+    @pytest.mark.parametrize("folders_enabled", [True, False])
+    @pytest.mark.parametrize("groups_enabled", [True, False])
+    @pytest.mark.parametrize("pages_enabled", [True, False])
+    def test_setting_scopes(
+        self, folders_enabled, groups_enabled, pages_enabled, pyramid_request
+    ):
```

Comment:
Combined here all of these.

The other ones were using a random assortment of naming for the parameters and fixutres.

Files and sections are different enought ot have their own tests.